src/NativeScript/inspector/GlobalObjectConsoleClient.cpp

```diff
@@ -115,12 +115,20 @@ void GlobalObjectConsoleClient::takeHeapSnapshot(JSC::ExecState*, const String&
 }
 
 void GlobalObjectConsoleClient::time(JSC::ExecState*, const String& title) {
-    m_consoleAgent->startTiming(title);
+    std::unique_ptr<String> startMsg = m_consoleAgent->startTiming(title);
+    const String* url;
+    if (startMsg) {
+        ConsoleClient::printConsoleMessage(MessageSource::ConsoleAPI, MessageType::Timing, MessageLevel::Warning, *startMsg.get(), (const String&)url, -1, -1);
+    }
 }
 
 void GlobalObjectConsoleClient::timeEnd(JSC::ExecState* exec, const String& title) {
-    RefPtr<Inspector::ScriptCallStack> callStack(Inspector::createScriptCallStackForConsole(exec, 1));
-    m_consoleAgent->stopTiming(title, WTFMove(callStack));
+    Ref<Inspector::ScriptCallStack> callStack(Inspector::createScriptCallStackForConsole(exec, 1));
+    std::unique_ptr<String> stopMsg = m_consoleAgent->stopTiming(title, WTFMove(callStack));
+    const String* url;
```

Comment:
`url` should be declared as `String url;` and can be moved inside the `if` block as it is not used outside. Also the cast to `const String &` in the call to `printConsoleMessage` should be removed as well.

Reply:
Same here,  you can pass `WTF::emptyString()` without having to construct a new instance.